app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -67,71 +192,45 @@ export default function DifferentialExpressionPanel({
+      {genesToShow &&
       <>
-        15 most DE genes
-        <a className="de-download-button"
-          onClick={async () => {await downloadBucketFile(bucketId, deFilePath)}}
-          data-analytics-name="differential-expression-download"
-          data-toggle="tooltip"
-          data-original-title="Download all DE genes data for this group"
-        >
-          <FontAwesomeIcon icon={faDownload}/>
-        </a>
-
-        <DifferentialExpressionModal />
-
-        <table data-testid="differential-expression-table" className="table table-terra table-scp-compact">
-          <thead>
-            <tr>
-              <th>Name</th>
-              <th>
-                <span className="glossary" data-toggle="tooltip" data-original-title="Log (base 2) of fold change">
-                  log<sub>2</sub>(FC)
-                </span>
-              </th>
-              <th>
-                <span className="glossary" data-toggle="tooltip" data-original-title="p-value adjusted with Benjamini-Hochberg FDR correction">
-                  Adj. p-value
-                </span>
-              </th>
-            </tr>
-          </thead>
-          <tbody>
-            {deGenes.map((deGene, i) => {
-              return (
-                <tr key={i}>
-                  <td>
-                    <label
-                      title="Click to view gene expression.  Arrow down (↓) and up (↑) to quickly scan."
-                    ><input
-                        type="radio"
-                        checked={checked[deGene.name]}
-                        data-analytics-name="selected-gene-differential-expression"
-                        value={deGene.name}
-                        onClick={event => {
-                          searchGenes([deGene.name])
-
-                          // Log this search to Mixpanel
-                          const speciesList = exploreInfo?.taxonNames
-                          const rank = i
-                          logSearchFromDifferentialExpression(
-                            event, deGene, speciesList, rank,
-                            clusterName, annotation.name
-                          )
-
-                          changeRadio(event)
-                        }}/>
-                      {deGene.name}</label></td>
-                  <td>{deGene.log2FoldChange}</td>
-                  <td>{deGene.pvalAdj}</td>
-                </tr>)
-            })}
-          </tbody>
-        </table>
-        <a href="https://forms.gle/qPGH5J9oFkurpbD76" target="_blank" title="Take a 1 minute survey">
-          Help improve this new feature
-        </a>
+        <div className="de-search-box">
+          <span className="de-search-icon">
+            <FontAwesomeIcon icon={faSearch} />
+          </span>
+          <input
+            className="de-search-input no-border"
+            name="de-search-input"
+            type="text"
+            autoComplete="off"
+            placeholder="Find gene"
```

Comment:
Would "Find a gene" make more sense here to help suggest that it's only for searching for a single gene?

Reply:
Nice refinement!  Done in 4452fb5.